packages/hs-react-ui/src/components/Table/Table.tsx

```diff
@@ -151,31 +190,74 @@ const SortIcon = styled(Icon)`
   `}
 `;
 
+/** Start of variables */
+
+const defaultCollapsed: collapsedState = {};
+const collapseIconColumn = {
+  name: '',
+  sortable: false,
+  width: '1rem',
+};
+
+// Keyboard listener required for a11y on the clickable span
+const onKeyPress = (evt: React.KeyboardEvent<HTMLSpanElement>) => {
+  if (evt.key === 'Enter' || evt.key === ' ') {
+    evt.preventDefault();
+    (evt.target as any).click();
+  }
+};
+
+// Supporting component used for the collapse icon by default in the Table
+const ExpansionIcon: React.FunctionComponent<ExpansionIconProps> = ({
+  collapsedIcon,
+  expandedIcon,
+  isCollapsed,
+  onClick,
+}: ExpansionIconProps) => {
+  const path = isCollapsed ? collapsedIcon : expandedIcon;
+  return (
+    <span tabIndex={0} onClick={onClick} role="button" onKeyPress={onKeyPress}>
+      <Icon path={path} size="1rem" />
```

Comment:
Let's default to Icon but we should let them pass any component for collapsed/expanded - we don't want to force mdi on the end developer

Reply:
This is just the default Icon used. The implementation allows for a completely custom component instead that doesn't need to be material, but it does need to be React by using the `expansionIconComponent` prop :)

Reply:
Lit